e2e/cypress/integration/unified/eth-rinkeby.js

```diff
@@ -8,5 +8,102 @@ describe('Unified - Ethereum Rinkeby', () => {
       expect(response.body).to.have.property('privateKey')
     })
   })
+  it('Get wallet', () => {
+   cy.request('GET', `${Cypress.config('baseUrl')}/api/v1/wallets?chain=ethereum&network=rinkeby&address=0xCDd0F0eE873195dfed20fc8AE7b11860CaE1799c&apiKey=58fc282a-498f-4695-88ac-f694fa98df9d`)
```

Comment:
You may use the [qs](https://www.npmjs.com/package/qs) for readability with these long query strings.